components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -253,4 +278,28 @@ std::string ViewCounterService::GetSuperReferralCode() const {
   return service_->GetSuperReferralCode();
 }
 
+void ViewCounterService::UpdateP3AValues() const {
+  uint64_t new_tab_count = new_tab_count_state_->GetHighestValueInWeek();
+  constexpr int kNewTabCount[] = {0, 1, 4, 9, 21, 51, 101};
+  const int* it_count =
+      std::lower_bound(kNewTabCount, std::end(kNewTabCount), new_tab_count);
+  int answer = it_count - kNewTabCount;
+  UMA_HISTOGRAM_EXACT_LINEAR("Brave.NTP.NewTabsCreated", answer,
+      7);
```

Comment:
turned out, lower_bound is a bit tricky than expected. I did a tes and apparently the right thing (according to the linked ticket) would be
`constexpr int kNewTabCount[] = {0, 3, 8, 20, 50, 100};`
and using `std::size(kNewTabCount) + 1` instead of 7

Reply:
also spacing can be improved